perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -118,18 +116,19 @@ sub munge_report_new_bodies {
         my $c = $self->{c};
         my $he = FixMyStreet::Cobrand::HighwaysEngland->new({ c => $c });
         my $on_he_road = $c->stash->{on_he_road} = $he->report_new_is_on_he_road;
-
+        my $on_he_road_for_litter;
         if (!$on_he_road) {
+            $on_he_road_for_litter = $c->stash->{on_he_road_for_litter} = $he->report_new_is_on_he_road_for_litter;
```

Comment:
Most (all?) of the litter roads overlap with the existing HE roads, so this check doesn't seem to happen at the right time.

<img width="860" alt="image" src="https://user-images.githubusercontent.com/4776/132534659-c98e222e-b1fb-47e3-be73-1158b17bfffc.png">


I think the check for litter roads should happen before the HE road check?